src/helpers/ignoreFilesPattern.ts

```diff
@@ -23,7 +23,7 @@ export const getFilteredSiteFiles = (site: Pick<Site, 'path'>) => {
 
 // returns the path to the site specific ignore file for the site backup
 // handles copying the default ignore file into the site prior to site backup
-export const getIgnoreFilePath = async (site: Site) => {
+export const getIgnoreFilePath = (site: Site) => {
```

Comment:
Removing `async` makes it a lot easier to call this function in the React component.